api/package-lock.json

```diff
@@ -0,0 +1,5969 @@
+{
```

Comment:
Can we make sure we're using Yarn for all package management? Looks like you might have used NPM here.